From 8811a0924567b573ec0825d8f9b0264e74435ae9 Mon Sep 17 00:00:00 2001
From: Ankit Agarwal <ankit.agarwal@intel.com>
Date: Wed, 12 Jun 2024 16:50:48 +0530
Subject: [PATCH] [WA] Fix for FILE_NAME_TAG

Getting build error- FILE_NAME_TAG is obsolete.
Replacing it with intel keyword for now.

Tracked-On: NA
Signed-off-by: Ankit Agarwal <ankit.agarwal@intel.com>
---
 tasks/flashfiles.mk | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/tasks/flashfiles.mk b/tasks/flashfiles.mk
index 297320c..b98f0a0 100755
--- a/tasks/flashfiles.mk
+++ b/tasks/flashfiles.mk
@@ -6,11 +6,11 @@ ifeq ($(TARGET_BUILD_TYPE),debug)
   name := $(name)_debug
 endif
 ifeq ($(RELEASE_BUILD),true)
-flash_name := $(name)-sign-flashfiles-$(FILE_NAME_TAG)
-target_name := $(name)-sign-targetfile-$(FILE_NAME_TAG)
+flash_name := $(name)-sign-flashfiles-intel
+target_name := $(name)-sign-targetfile-intel
 gpt_name := $(PRODUCT_OUT)/release_sign/$(name).img
 endif
-name := $(name)-flashfiles-$(FILE_NAME_TAG)
+name := $(name)-flashfiles-intel
 BUILDNUM := $(shell $(DATE) +%H%M%3S)
 ifeq ($(RELEASE_BUILD),true)
 BUILT_RELEASE_FLASH_FILES_PACKAGE := $(PRODUCT_OUT)/$(flash_name).zip
@@ -115,7 +115,7 @@ ifneq ($(FLASHFILE_VARIANTS),)
       $(foreach var,$(FLASHFILE_VARIANTS), \
 	    $(info Adding $(var)) \
 	    $(eval fn_prefix := $(PRODUCT_OUT)/$(TARGET_PRODUCT)) \
-	    $(eval fn_suffix := $(var)-$(FILE_NAME_TAG)) \
+	    $(eval fn_suffix := $(var)-intel) \
 	    $(eval ff_zip := $(fn_prefix)-flashfiles-$(fn_suffix).$(fn_compress_format)) \
 	    $(eval INTEL_FACTORY_FLASHFILES_TARGET += $(ff_zip)) \
 	    $(call dist-for-goals,droidcore,$(ff_zip):$(notdir $(ff_zip))))
@@ -135,7 +135,7 @@ ifneq ($(TARGET_SKIP_OTA_PACKAGE), true)
   $(foreach var,$(OTA_VARIANTS), \
 	$(info Adding $(var)) \
 	$(eval fn_prefix := $(PRODUCT_OUT)/$(TARGET_PRODUCT)) \
-	$(eval fn_suffix := $(var)-$(FILE_NAME_TAG)) \
+	$(eval fn_suffix := $(var)-intel) \
 	$(eval ota_zip := $(fn_prefix)-ota-$(fn_suffix).zip) \
 	$(eval INTEL_OTA_PACKAGES += $(ota_zip)) \
 	$(call dist-for-goals,droidcore,$(ota_zip):$(notdir $(ota_zip))))
@@ -314,9 +314,9 @@ publish_vertical:
 endif
 
 ifeq ($(RELEASE_BUILD),true)
-ISO_INSTALL_IMG = $(PRODUCT_OUT)/$(TARGET_PRODUCT)-sign-flashfile-$(FILE_NAME_TAG).iso
+ISO_INSTALL_IMG = $(PRODUCT_OUT)/$(TARGET_PRODUCT)-sign-flashfile-intel.iso
 else
-ISO_INSTALL_IMG = $(PRODUCT_OUT)/$(TARGET_PRODUCT)-flashfile-$(FILE_NAME_TAG).iso
+ISO_INSTALL_IMG = $(PRODUCT_OUT)/$(TARGET_PRODUCT)-flashfile-intel.iso
 endif
 ifeq ($(use_tar),true)
 ISO_INSTALL_IMG_COMP = $(ISO_INSTALL_IMG).tar.gz
-- 
2.34.1

